lib/rewriter.js

```diff
@@ -46,6 +47,13 @@ function rewriter(fileInfo) {
   sourceFile.forEachDescendant((node) => replaceGetSets(node, imports));
   addAllImports(sourceFile, imports);
 
+  // remove JSDocs
```

Comment:
Can you also add the TODO pointer to #71 here?

Reply:
Sure